common/metrics/config.go

```diff
@@ -194,6 +194,12 @@ var (
 			200,
 			500,
 			1000,
+			2000,
+			5000,
+			10000,
+			20000,
+			50000,
+			100000,
```

Comment:
We emit metrics for # of pending tasks in a shard, and suspend task loading if the number is too high. The original 1000 is too low and can't give a good estimation of the # of pending tasks nor can be used to tell if task loading is suspended or not.